code/datums/materials/basemats.dm

```diff
@@ -257,7 +257,7 @@ Unless you know what you're doing, only use the first three numbers. They're in
 	desc = "A weird kind of ice, feels warm to the touch"
 	color = "#88cdf1"
 	alpha = 150
-	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	categories = list(MAT_CATEGORY_NORECIPE = TRUE)
```

Comment:
this makes no sense. hot_ice IS rigid, just make a new flag thats called MAT_CATEGORY_SHEETCRAFTING and use that for the stackcrafting list instead.